### PR TITLE
BIZ-43: GitHub organization Readme

### PR DESCRIPTION
**Pull request by KhouloudBL**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Lombiq Technologies
```
Comment (Piedone) [inline profile/README.md:1]: You can view the file, as will be displayed on GitHub, here:

image

Check for any styling issues that aren't necessarily apparent from Markdown. What I immediately noticed here is that the logos are huge and distorted.

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+
+[Lombiq Technologies](https://lombiq.com) is a software, training, and services company focusing on web development with open Microsoft technologies. 
+Our main area of expertise is the Orchard Core content management system. We provide custom development, training, hosting, and consulting services and we are behind the first (and only) Orchard CMS SaaS.
+
+## Products:
+
+* [**Hastlayer**](https://hastlayer.com/) turns performance-critical sections of .NET programs into computer chips (FPGA). You can still update your software, but it will be a whole lot faster and more power-efficient.
+
+ <img src="https://user-images.githubusercontent.com/68704777/173390183-8c2e83ae-9ce3-47c4-a62e-4c58d83648e0.png" width="1000" height="250">
```
Comment (Piedone) [inline profile/README.md]: You can include images directly in Markdown too. Be sure to add an alt text to everything too.

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+* [**DotNest**](https://dotnest.com/) We've developed DotNest as a software-as-a-service for Orchard CMS, enabling users to create their websites using Orchard in a flash. Today it runs more than 6000 websites.
+
+
+<img src="https://user-images.githubusercontent.com/68704777/173390495-9c1fa7a2-084c-4a9b-bc6b-c59e183fb784.png" width="400" height="400">
```
Comment (Piedone) [inline profile/README.md]: Add suitable small (the physical dimensions, i.e. resolution of the images, not the display size) images directly to this repo so they can be loaded directly, not from something external.

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+# Lombiq Technologies 
+
+
+[Lombiq Technologies](https://lombiq.com) is a software, training, and services company focusing on web development with open Microsoft technologies. 
+Our main area of expertise is the Orchard Core content management system. We provide custom development, training, hosting, and consulting services and we are behind the first (and only) Orchard CMS SaaS.
```
Comment (Piedone) [inline profile/README.md]:
```suggestion
[Lombiq Technologies](https://lombiq.com) is a software, training, and services company focusing on web development with open Microsoft technologies. 
Our main area of expertise is the [Orchard Core content management system](https://www.orchardcore.net/). We provide custom development, training, hosting, and consulting services and we are behind the first (and only) Orchard CMS SaaS, [DotNest](https://dotnest.com).
```

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+ <img src="https://user-images.githubusercontent.com/68704777/173390183-8c2e83ae-9ce3-47c4-a62e-4c58d83648e0.png" width="1000" height="250">
+
+
+* [**DotNest**](https://dotnest.com/) We've developed DotNest as a software-as-a-service for Orchard CMS, enabling users to create their websites using Orchard in a flash. Today it runs more than 6000 websites.
+
+
+<img src="https://user-images.githubusercontent.com/68704777/173390495-9c1fa7a2-084c-4a9b-bc6b-c59e183fb784.png" width="400" height="400">
+ 
+## Our services:
```
Comment (Piedone) [inline profile/README.md]: Put this above Products.

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
+# Lombiq Technologies 
+
+
+[Lombiq Technologies](https://lombiq.com) is a software, training, and services company focusing on web development with open Microsoft technologies. 
+Our main area of expertise is the Orchard Core content management system. We provide custom development, training, hosting, and consulting services and we are behind the first (and only) Orchard CMS SaaS.
+
+## Products:
```
Comment (Piedone) [inline profile/README.md]: No need for colons at the end of titles.

File: profile/README.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+## Our services:
+
+### Software development:
+ 
+Our team's passion is developing software with care for clients who need help with their most extraordinary projects in the following frameworks:
+
+- Orchard Core.
+- Orchard CMS.
+- ASP.NET Core.
+- ASP.NET.
```
Comment (Piedone) [inline profile/README.md]:
```suggestion
- Orchard Core
- Orchard CMS
- ASP.NET Core
- ASP.NET
```

File: profile/README.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+**We also contribute a vast amount of code, documentation, and learning resources to the open-source ecosystem. These materials are also available for our partners to use even after we have delivered a successful project.**
+
+## Our team
+Lombiq's team consists of highly skilled experts from diverse backgrounds:
+
+ <img src="https://user-images.githubusercontent.com/68704777/173390541-bf4c0068-611b-4552-8983-99afb10f4acf.jpg" width="1400" height="700">
+
+
+## Open-source and open hearts
+Apart from the software world we also try to help in other communities with our philanthropic activities. Most recently, we've delivered thousands of reusable masks to the Hungarian Charity Service of the Order of Malta during the coronavirus pandemic, as well as help for refugees of the war in Ukraine, and again.
```
Comment (Piedone) [inline profile/README.md]: Link to https://lombiq.com/open-source.

File: profile/README.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+That would be really fantastic! If you've found one of our projects valuable and you'd like to contribute in any way please check out our [guidelines](https://lombiq.com/open-source-guidelines) and go ahead, we'd be very happy to welcome your efforts! Also, consider contributing, in-kind or otherwise, to the projects supported by us.
+
+You can also check our profile in [NuGet](https://www.nuget.org/profiles/Lombiq).
+
+Would you like to get in touch with us? Feel free to reach out to us through our Social media accounts:
+* [Twitter](https://twitter.com/Lombiq)
+* [YouTube](https://www.youtube.com/c/LombiqTechnologies)
+* [LinkedIn](https://www.linkedin.com/company/3162451/) 
+
+Or you can write us an email using the following link: [contact us](https://lombiq.com/contact-us).
```
Comment (Piedone) [inline profile/README.md]:
```suggestion
Or you can [get in touch with us here](https://lombiq.com/contact-us).
```

File: profile/README.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+
+## Open-source and open hearts
+Apart from the software world we also try to help in other communities with our philanthropic activities. Most recently, we've delivered thousands of reusable masks to the Hungarian Charity Service of the Order of Malta during the coronavirus pandemic, as well as help for refugees of the war in Ukraine, and again.
+
+# Would you like to contribute to our projects?
+
+That would be really fantastic! If you've found one of our projects valuable and you'd like to contribute in any way please check out our [guidelines](https://lombiq.com/open-source-guidelines) and go ahead, we'd be very happy to welcome your efforts! Also, consider contributing, in-kind or otherwise, to the projects supported by us.
+
+You can also check our profile in [NuGet](https://www.nuget.org/profiles/Lombiq).
```
Comment (Piedone) [inline profile/README.md]:
```suggestion
You can also check our profile on [NuGet](https://www.nuget.org/profiles/Lombiq) with packages for our open-source projects.
```

File: profile/README.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+
+* [**DotNest**](https://dotnest.com/) We've developed DotNest as a software-as-a-service for Orchard CMS, enabling users to create their websites using Orchard in a flash. Today it runs more than 6000 websites.
+
+
+<img src="https://user-images.githubusercontent.com/68704777/173390495-9c1fa7a2-084c-4a9b-bc6b-c59e183fb784.png" width="400" height="400">
+ 
+## Our services:
+
+### Software development:
```
Comment (Piedone) [inline profile/README.md]: With small code formatting things like line breaks, please follow https://orcharddojo.net/orchard-resources/CoreLibrary/ContributionGuideLines.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 3 above)
Comment (KhouloudBL) [reply profile/README.md]: Hastlayer logo
Is this kind of size okay? Can I add this picture as it is ?

Comment (KhouloudBL) [reply profile/README.md]: Another option would be this. I hope one of them is right :)
Hastlayer_logo_50

Comment (Piedone) [reply profile/README.md]: Yeah, seems about right. Just be sure to keep the transparency of the PNG.

Comment (KhouloudBL) [reply profile/README.md]: I am not sure if I got what you mean but I will submit the updated version with the new sized pics and you let me know if there are any further changes needed. :)

Comment (Piedone) [reply profile/README.md]: When resizing images, keep the aspect ratios.

File: profile/README.md (hunk 6 above)
Comment (Piedone) [reply profile/README.md]: This isn't addressed.

File: profile/README.md (hunk 8 above)
Comment (Piedone) [reply profile/README.md]: Perhaps not from the title though :). Add one in the text, you needn't keep the current text.

File: profile/README.md (hunk 10 above)
Comment (Piedone) [reply profile/README.md]: Next time commit such suggestions from under the PR, please, so they can be auto-resolved.

File: profile/README.md [hunk 12]
```diff
@@ -1,65 +1,66 @@
...
 
 ### Consulting:
-
 We offer Orchard consulting in both Orchard editions, Orchard Core and Orchard CMS, including website and application development, and code review.
 
 ### Training:
-
 Our team of dedicated experts is here to help groups and individuals who want to learn about Orchard or simply broaden their coding knowledge. Our [Orchard Dojo](https://orcharddojo.net/) website contains several training sessions to help beginners become experts.
 
+![Orchard Dojo](https://user-images.githubusercontent.com/68704777/174299406-a7ea1222-a1a8-457e-b68f-e3570bf2d5b6.png)
```
Comment (Piedone) [inline profile/README.md]: Let's leave out these logos, actually, because they look bad in dark mode:

image

File: profile/README.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+* [**DotNest**](https://dotnest.com/) We've developed DotNest as a software-as-a-service for Orchard CMS, enabling users to create their websites using Orchard in a flash. Today it runs more than 6000 websites.
+
+ ![DotNest logo medium](https://user-images.githubusercontent.com/68704777/174301139-3c4ab813-8fe8-4ca1-a428-d9bf4cd98bdb.png)
+
+
+## Our team
+
+Lombiq's team consists of highly skilled experts from diverse backgrounds:
+ 
+ ![lombiq team 2021 summer](https://user-images.githubusercontent.com/68704777/174300662-93d75ff3-5c7d-48c4-9236-775634402287.jpg)
```
Comment (Piedone) [inline profile/README.md]: You should be able to reference this image directly with the path "lombiq-team-2021-summer_50.jpg". Also, rather name the file just "lombiq-team-2021-summer.jpg"

**Review by Piedone — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 6 above)
Comment (Piedone) [reply profile/README.md]: This still isn't addressed.

File: profile/README.md [hunk 14]
```diff
@@ -32,23 +30,21 @@ Our team of dedicated experts is here to help groups and individuals who want to
...
  
- ![lombiq team 2021 summer](https://user-images.githubusercontent.com/68704777/174300662-93d75ff3-5c7d-48c4-9236-775634402287.jpg)
+ <img src="lombiq-team-2021-summer_50.jpg" alt="lombiq team 2021 summer" width="1000"  height="700">
 
 
-## [Open source](https://lombiq.com/open-source) and open hearts
+## Open source and open hearts
 
-Apart from the software world we also try to help in other communities with our philanthropic activities. Most recently, we've delivered thousands of reusable masks to the Hungarian Charity Service of the Order of Malta during the coronavirus pandemic, as well as help for refugees of the war in Ukraine, and again.
+Apart from the [Open source](https://lombiq.com/open-source) software world, we also try to help in other communities with our philanthropic activities. Most recently, we've delivered thousands of reusable masks to the Hungarian Charity Service of the Order of Malta during the coronavirus pandemic, as well as help for refugees of the war in Ukraine, and again.
```
Comment (Piedone) [inline profile/README.md]:
```suggestion
Apart from [our activities in the open-source world](https://lombiq.com/open-source), we also try to help other communities with our philanthropic activities. Most recently, we've delivered thousands of reusable masks to the Hungarian Charity Service of the Order of Malta during the coronavirus pandemic, as well as help for refugees of the war in Ukraine, and again.
```

File: profile/README.md [hunk 15]
```diff
@@ -22,8 +22,6 @@ We offer Orchard consulting in both Orchard editions, Orchard Core and Orchard C
 ### Training:
 Our team of dedicated experts is here to help groups and individuals who want to learn about Orchard or simply broaden their coding knowledge. Our [Orchard Dojo](https://orcharddojo.net/) website contains several training sessions to help beginners become experts.
 
-![Orchard Dojo](https://user-images.githubusercontent.com/68704777/174299406-a7ea1222-a1a8-457e-b68f-e3570bf2d5b6.png)
```
Comment (Piedone) [inline profile/README.md]: Remove the now unneeded files from the repo.

File: profile/README.md [hunk 16]
```diff
@@ -32,23 +30,21 @@ Our team of dedicated experts is here to help groups and individuals who want to
...
- ![DotNest logo medium](https://user-images.githubusercontent.com/68704777/174301139-3c4ab813-8fe8-4ca1-a428-d9bf4cd98bdb.png)
-
 
 ## Our team
 
 Lombiq's team consists of highly skilled experts from diverse backgrounds:
+
  
- ![lombiq team 2021 summer](https://user-images.githubusercontent.com/68704777/174300662-93d75ff3-5c7d-48c4-9236-775634402287.jpg)
+ <img src="lombiq-team-2021-summer_50.jpg" alt="lombiq team 2021 summer" width="1000"  height="700">
```
Comment (Piedone) [inline profile/README.md]: Why did you change back to an `img` tag when we discussed that you shouldn't use them? Why are there two images in the repo now? We need a single image, which has the correct size and aspect ratio, named lombiq-team-2021-summer.jpg, in Markdown, with an alt text that has correct spelling.

File: profile/README.md [hunk 17]
```diff
@@ -32,23 +30,21 @@ Our team of dedicated experts is here to help groups and individuals who want to
...
 
 
-## [Open source](https://lombiq.com/open-source) and open hearts
+## Open source and open hearts
```
Comment (Piedone) [inline profile/README.md]:
```suggestion
## Open-source and open hearts
```

**Review by Piedone — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 16 above)
Comment (KhouloudBL) [reply profile/README.md]: Because as I mentioned in the discussion, I cannot add the aspect ratios Zoltan. I tried the 3 following ways mentioned here: 
https://www.youtube.com/watch?v=f4ifdKCH7VI
https://www.youtube.com/watch?v=hHbWF1Bvgf4
 https://www.youtube.com/watch?v=teGy35UGnds
 Same goes for the websites I checked like: 
 https://stackoverflow.com/questions/24383700/resize-image-in-the-wiki-of-github-using-markdown
 and here as well: 
 https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

 The format introduced is the one I used. Other than that, it won't work for me. Please let me know if there is any other source I can work with. And again, thank you for your patience. 
 
 PS: I checked the video tutorial as well, but he didn't have one specific for Picture resizing and aspect ratios. So, I am still checking.
 
error

 Here you can see that only the first one worked: 
error1

Comment (Piedone) [reply profile/README.md]: When talking about resizing and aspect ratios, I'm talking about resizing the file, what you already did before. If the file is the correct size, then you don't need to set anything in HTML.

File: profile/README.md (hunk 6 above)
Comment (KhouloudBL) [reply profile/README.md]: I'd like to ask . What do you mean by "isn't addressed"?

Comment (Piedone) [reply profile/README.md]: I asked for the colons at the end of titles to be removed. There are still colons at the end of titles, and you didn't disagree with my request either.

File: profile/README.md (hunk 16 above)
Comment (KhouloudBL) [reply profile/README.md]: I did it last time and you asked why I removed the aspect ratios. That's why I changed it back with the resized file. I misunderstood you then.
So, for now, I must take the team picture (the resized one) and just copy paste it from the repo to the Markdown file. No height width aspects are needed. Is that it?

Comment (Piedone) [reply profile/README.md]: What I said is that images were distorted, because you did resizing without maintaining the aspect ratios. The resizing itself should maintain aspect ratios, not client-side resizing in HTML.

You need to put an appropriately resized file into the repo, as you already did, and reference it from Mardown with the Markdown, not HTML code, what you also did before.

Comment (KhouloudBL) [reply profile/README.md]: I think I understand you better now. Thank you for clarifying your points. I'll submit an updated version right away. I will use the resized image from the repo. I hope this time I get it correct fully :)

File: profile/README.md [hunk 18]
```diff
@@ -39,12 +39,12 @@ Our team of dedicated experts is here to help groups and individuals who want to
 Lombiq's team consists of highly skilled experts from diverse backgrounds:
 
  
- <img src="lombiq-team-2021-summer_50.jpg" alt="lombiq team 2021 summer" width="1000"  height="700">
+ ![lombiq team 2021 summer](lombiq-team-2021-summer.jpg)
```
Comment (Piedone) [inline profile/README.md]: Again, please use correct spelling in the alt text. The company's name is not "lombiq".

**Review by Piedone — CHANGES_REQUESTED** [review]
